mowgli_etl/pipeline/wdc/wdc_transformer.py

```diff
@@ -141,7 +141,7 @@ def transform(
                     if listing == None:
                         listing = category
 
-                product = HPTC().classify(title=listing)
+                product = HPTC.classify(title=listing)
```

Comment:
Use @staticmethod or @classmethod if you're going to do this.

Reply:
The classifier instance should be a parameter to the transformer, in any case, so you'll want to make normal method calls to the instance here. So no static/classmethod.